<launch>

  <arg name="identity" default="default" />

  <node pkg="memory" type="animations_manager.py" name="animations_manager">
    <param name="id_robot" value="1" />
  	<param name="nao_ip" value="10.0.0.10" />
  	<param name="nao_port" value="9559" />
  	<param name="periodUpdateTimerBSI" value="0.1" />  
  	<param name="topic_animations" value="topic_animations" />
  	<param name="topic_settings" value="topic_settings" />
  	<param name="topic_poses" value="topic_poses" />
  	<param name="topic_activity" value="topic_activity" />
  	<param name="topic_end_animation" value="topic_end_animation" />
  	<param name="parkinson_scale" value="3" />
  </node>

  <!--
  <node pkg="memory" type="animations_manager.py" name="animations_manager2">
  	<param name="id_robot" value="2" />
  	<param name="nao_ip" value="10.0.0.14" />
  	<param name="nao_port" value="9559" />
  	<param name="periodUpdateTimerBSI" value="0.1" />  
  	<param name="topic_animations" value="topic_animations" />
  	<param name="topic_settings" value="topic_settings" />
  	<param name="topic_poses" value="topic_poses" />
  	<param name="topic_activity" value="topic_activity" />
  	<param name="topic_end_animation" value="topic_end_animation" />
  	<param name="parkinson_scale" value="3" />
  </node>
  -->

  <node pkg="memory" type="MemoryGame.py" name="memory_game">
    <param name="topic_activity" value="topic_activity" />
    <param name="topic_publisher_tablet" value="chatter" />
    <param name="topic_listener_tablet" value="game_action" />
    <param name="topic_end_animation" value="topic_end_animation" />
    <param name="topic_gaze_match" value="topic_gaze_match" />
    <param name="topic_end_record" value="topic_end_record" />
    <param name="time_between_moves" value="5" />

    <param name="game_number" value="0"/>
    <param name="identity" value="$(arg identity)"/>
  </node>

  <!-- Start the feature face tracker -->
  <include file="$(find features_face)/launch/features_face.launch">
    <param name="identity" value="$(arg identity)"/>
  </include>

  <!-- Start the camera/audio reccord -->
  <!--
  <include file="$(find record_audio_video)/launch/record_launch.launch">
    <arg name="identity" value="$(arg identity)"/>
  </include>
  -->

</launch>